tests/integration/devfile/cmd_devfile_catalog_test.go

```diff
@@ -79,4 +79,10 @@ var _ = Describe("odo devfile catalog command tests", func() {
 			helper.MatchAllInOutput(output, []string{"name: nodejs-starter", "Registry: " + registryName})
 		})
 	})
+
+	Context("When checking catalog for installed services", func() {
+		It("should succeed", func() {
+			helper.CmdShouldPass("odo", "catalog", "list", "services")
```

Comment:
@dharmit Not sure, how does it(passing checks) verify the functionality of catalog list services. Shouldn't we validate with the installed services?

Reply:
@prietyc123 this is `odo catalog list services`. What you are talking about is `odo service list`.
 `odo catalog list services` - list all services available on the cluster that odo can create
 `odo service list`  -  list services that were already created